docs/write_case.md

```diff
@@ -1,64 +1,300 @@
+
+Please keep in mind that your code may be copied by others.
+
+### Compositions of test
+
+The LISA test is composed of metadata, setting/cleaning up, and test body.
+
+### Metadata
+
+Metadata is used to provide documentation and the settings of test suites and test cases.
+
+#### Test suite metadata
+
+- **area** classifies test suites by belonging. When it needs to have a special validation on some area, it can be used to filter test cases. The values can be provisioning, cpu, memory, storage, network, etc.
+- **category** categorizes test cases by test type. It includes functional, performance, pressure, and community. Performance and stress test cases take longer to run, which is not included in regular operations. Community test cases are wrappers that help provide results comparable to the community.
+- **description** should introduce this test suite, including purpose, coverage, and any other content, which is helpful to understand this test suite.
+- **name** is optional. The default name is the class name and can be replaced with the name field. The name is part of the test name. Just like the name space in a programming language.
+- **requirement** defines the default requirements for this test suite and can be rewritten at the test case level. Learn more from [concepts](concepts.md).
+
+See [examples](../examples/testsuites) for details.
+
+```python
+@TestSuiteMetadata(
+    area="provisioning",
+    category="functional",
+    description="""
+    This test suite uses to verify if an environment can be provisioned correct or not.
+
+    - The basic smoke test can run on all images to determinate if a image can boot and
+    reboot.
+    - Other provisioning tests verify if an environment can be provisioned with special
+    hardware configurations.
+    """,
+)
+class Provisioning(TestSuite):
+    ...
+```
+
+#### Test case metadata
+
+- **priority** priority depends on the impact of the test case and is used to determine how often to run the case. Learn more from [concepts](concepts.md).
+- **description** explains the purpose and procedures of the test. It is used to generate test specification documents.
+- **requirement** define the requirements in this case. If no requirement is specified, the test suite or global default requirements will be used.
+
+See [examples](../examples/testsuites) for details.
+
+```python
+@TestCaseMetadata(
+    description="""
+    This case verifies whether a node is operating normally.
+
+    Steps,
+    1. Connect to TCP port 22. If it's not connectable, failed and check whether
+        there is kernel panic.
+    2. Connect to SSH port 22, and reboot the node. If there is an error and kernel
+        panic, fail the case. If it's not connectable, also fail the case.
+    3. If there is another error, but not kernel panic or tcp connection, pass with
+        warning.
+    4. Otherwise, fully passed.
```

Comment:
See comments at the other commit. Can md files include excerpts from other files? That would be ideal.

Reply:
It's an example, and for reference here. I will enable doc strings and combine with document in later tasks.